SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp

```diff
@@ -74,6 +75,19 @@ bool UGeneratedTestMap::GenerateCustomConfig()
 		// clang-format on
 	}
 
+	// If true, disable the RunUnderOneProcess setting
+	if (bEnableMultiProcess == true)
+	{
+		if (!CustomConfigString.IsEmpty())
+		{
+			CustomConfigContent += LINE_TERMINATOR;
```

Comment:
I guess this following the style already there, but yikes..

Reply:
tbf, I think both this one and the above can actually just always start with a newline, if that is the problem here

Reply:
As I said elsewhere, would prefer a vector of entries joined at point of use, but maybe this change is out of scope.

Reply:
hmm, don't think that's possible here, though what we could maybe do at some point in the future is to reuse Unreal's machinery for writing config files

Reply:
Since it's being re-used, perhaps a function?

Reply:
I don't think a function would be needed, just two things of this style
```
CustomConfigContent += FString::Printf(LINE_TERMINATOR TEXT("[/Script/UnrealEd.LevelEditorPlaySettings]") LINE_TERMINATOR
			TEXT("RunUnderOneProcess=false"));
```
formatted more nicely than I did there